src/picongpu/include/fields/FieldJ.kernel

```diff
@@ -342,7 +342,14 @@ struct KernelAddCurrentToEMF
         const DataSpace<simDim > threadIndex(threadIdx);
         auto fieldJBlock = fieldJ.shift(blockCell);
 
-        ThreadCollective<BlockArea> collective(threadIndex);
+        ThreadCollective<
+            BlockArea,
+            PMacc::math::CT::volume< typename BlockArea::SuperCellSize >::type::value
+        > collective(
+            DataSpaceOperations< BlockArea::Dim >::template map<
+                typename BlockArea::SuperCellSize
+            >( threadIndex )
```

Comment:
I don't get it. In the old code you just passed the `threadIndex` but now a map. However the constructor https://github.com/psychocoderHPC/picongpu/blob/cfd1322c56f4e953fa2d6b96a9e5ea97fc7a55f0/src/libPMacc/include/mappings/threads/ThreadCollective.hpp#L71 did not (?) change.
So why?